backend/src/main/java/de/amos/apachepulsarui/dto/ConsumerDto.java

```diff
@@ -41,4 +43,14 @@ public static ConsumerDto create(ConsumerStats consumerStats) {
                 .blockedConsumerOnUnackedMsgs(consumerStats.isBlockedConsumerOnUnackedMsgs())
                 .build();
     }
+
+
+    private static ConsumerStats getConsumerStatsByTopic(TopicStats topicStats, String consumer) {
+        return topicStats.getSubscriptions()
+                .values().stream()
```

Comment:
This is the way I prefer procedural method chains to be formatted, I'd just prefer if we do it consistently everywhere 😄